packages/katello/rubygem-pulp_2to3_migration_client/rubygem-pulp_2to3_migration_client.spec

```diff
@@ -4,17 +4,17 @@
 
 %global gem_name pulp_2to3_migration_client
 
-%global release 1
-%global prerelease rc1
+%global release 1	
+%global prerelease b1	
 %global prereleaserpm %{?prerelease:.}%{?prerelease}
 
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.0.1
 Epoch: 1
-Release: %{?prereleaserpm:0.}%{release}%{?prereleaserpm}%{?dist}
-Summary: Pulp 2 to Pulp 3 API Ruby Gem
+Version: 0.2.0
+Release: 1%{?dist}
```

Comment:
@evgeni : Does this look right to you? Or should I just keep 
` %{?prereleaserpm:0.}%{release}%{?prereleaserpm}%{?dist}`